src/utils/style-utils.js

```diff
@@ -48,3 +81,19 @@ export const fontFamilyOptions = FONT_FAMILIES.map((font) => ({
   value: font,
   label: getFirstFont(font),
 }));
+
+export const getColor = (
```

Comment:
since there is a color-utils, doesn't this fit better there?

Reply:
I found color-utils it more related to set css colors and color code stuff... but sure I can move it over there.